inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -108,6 +108,19 @@ static const char layout_message[] = "Optional. Prompts how network layouts shou
 // @brief message for quantization bits
 static const char gna_qb_message[] = "Optional. Weight bits for quantization:  8 or 16 (default)";
 
+// TODO: duplicate options from compile_tool
```

Comment:
Do we need this TODO in benchmark_app header?

Reply:
Removed.